### DISCUSSION TITLE
Oversaturated yellows in oupaint with certain checkpoint models

### DISCUSSION THREAD

**Discussion by chenav** [lllyasviel/Fooocus#3261] [Q&A] [answered]
Hello, 
I am using nightvision checkpoint models ( https://civitai.com/models/128607/nightvisionxl), until version 8, everything is ok when used in Fooocus.
But with the new version 9, outpaint will **always** generate yellow outpaint zones. 

Here are examples:
![image](https://github.com/lllyasviel/Fooocus/assets/16047594/db1f1c60-37e9-4792-85ee-ce29621aebf1)
v8 zoom out in fooocus

![image](https://github.com/lllyasviel/Fooocus/assets/16047594/56d2f8bf-cd89-460f-973b-ec5a8450deb9)
v9 zoom out in foocus 

It is always the case, even with B&W or "cold colors" oriented images.

![image](https://github.com/lllyasviel/Fooocus/assets/16047594/6ce96b99-9235-418b-99bf-0f08c4b5e07b)
"cold colors" zoom out

Where does that come from ? Is it possible to counter this behaviour with specific settings ?

Thanks !

C.

**Comment by mashb1t** [comment]
Ensure that v9 has a VAE include and if it doesn't, download and select one yourself. Also try to set the inpaint engine to none, might help but not certain.

**Reply by chenav** [reply, marked as answer]
VAE is baked in the checkpoint model. Even though, I tried to add a VAE : it did not change the results.
I tried also with or without inpaint engines (all versions). I played with CFG and ADM positive/negative paramaters. 
I always get the yellowish frame...
Something is "wrong" in the v9 model. 
I am going to ask their team about that "feature"
